test/e2e/persistent_volumes-gce.go

```diff
@@ -107,11 +107,6 @@ var _ = framework.KubeDescribe("PersistentVolumes:GCEPD [Volume]", func() {
 			framework.DeletePodWithWait(f, c, clientPod)
 			framework.PVPVCCleanup(c, ns, pv, pvc)
 			clientPod, pv, pvc, node = nil, nil, nil, ""
-		}
-	})
-
-	AddCleanupAction(func() {
-		if len(diskName) > 0 {
```

Comment:
1) why don't you need this `len` check in the `AfterEach` above?
2) should diskname be set to "" somewhere?